src/utils/findVideoInWatchlater.js

```diff
@@ -0,0 +1,3 @@
+export const findVideoInWatchlater = (videoId, watchlater) => {
+  return watchlater.filter((video) => video._id === videoId)[0];
+};
```

Comment:
here you can use `.find()` method instead of `.filter()`

```js
export const findVideoInWatchlater = (videoId, watchlater) => 
  watchlater.find((video) => video._id === videoId);
```